test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlCommandTests.cs

```diff
@@ -100,21 +100,26 @@ private void SubmitsTracesWithNetStandard(string packageVersion, bool enableCall
             // Once this is implemented, this will add another 1 group for the direct assembly reference
             // and another 1 group for the netstandard assembly reference
 #if NET452
-            var expectedSpanCount = 50; // 7 queries * 7 groups + 1 internal query
-#else
             var expectedSpanCount = 78; // 7 queries * 11 groups + 1 internal query
+#elif NET461
+            var expectedSpanCount = 134;
+#else
+            var expectedSpanCount = 137; // 7 queries * 19 groups + 1 internal query
+#endif
+
```

Comment:
Update the comments to reflect the numbers?

Reply:
I put off updating these 😅 Done in f9829db87629aa5c357cc4ad5a69a51fcd1da1ad